pkg/server/server.go

```diff
@@ -92,6 +92,12 @@ var (
 		NativeHistogramMaxBucketNumber:  100,
 		NativeHistogramMinResetDuration: time.Hour,
 	}, []string{"grpc_service", "grpc_method", "datastore_query_count", "dispatch_count", "consistency"})
+
+	throttledRequestCounter = promauto.NewCounter(prometheus.CounterOpts{
+		Namespace: build.ProjectName,
+		Name:      "throttled_requests_count",
+		Help:      "The total number of requests that have been throttled.",
+	})
```

Comment:
```suggestion
	throttledRequestCounter = promauto.NewCounter(prometheus.CounterOpts{
		Namespace: build.ProjectName,
		Name:      "throttled_requests_count",
		Help:      "The total number of requests that have been throttled.",
	}, []string{"grpc_service", "grpc_method"})
```

Each API can use their own throttling settings, so it's best to get data for each API individually